src/main/java/com/checkmarx/flow/service/GitHubService.java

```diff
@@ -8,12 +8,7 @@
 import com.checkmarx.flow.config.GitHubProperties;
 import com.checkmarx.flow.config.ScmConfigOverrider;
 import com.checkmarx.flow.constants.FlowConstants;
-import com.checkmarx.flow.dto.OperationResult;
-import com.checkmarx.flow.dto.RepoComment;
-import com.checkmarx.flow.dto.RepoIssue;
-import com.checkmarx.flow.dto.ScanDetails;
-import com.checkmarx.flow.dto.ScanRequest;
-import com.checkmarx.flow.dto.Sources;
+import com.checkmarx.flow.dto.*;
 import com.checkmarx.flow.dto.github.Content;
```

Comment:
What are the changes in this file? not clear

Reply:
optimization from IDEA